packages/api-derive/src/balances/index.ts

```diff
@@ -2,6 +2,8 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
+export * from './all';
+export * from './allMulti';
```

Comment:
Not crazy about the `allMulti` naming here. :( However, I'm complaining without a better suggestion, so probably need to live with it.

Reply:
On second though, let's drop the multi. (We are not using it internally and hopefully the API will have native multi call support real-soon-now - it is my top API priority to do)